lib/Client.php

```diff
@@ -48,7 +58,7 @@ public function send(RequestInterface $apiRequest)
         try {
             $response = $this->client->send(
                 $request,
-                ['timeout' => $this->timeout]
+                $this->requestOptions
```

Comment:
Why the old way wasn't working?

Reply:
I this method, it was working but, as I said above, the `timeout` property was deprecated and now, the client has `$requestOptions` informed to `send` method.

Reply:
I got it, thanks for your explanation. =)